next.clayui.com/content/docs/components/forms/multi-step-forms.mdx

```diff
@@ -0,0 +1,70 @@
+---
```

Comment:
I think it's more valuable to add information from this component and how to use it, just adding examples might not be a good idea since Storybook does the same.